target/linux/lantiq/patches-4.19/4053-NET-reactivate-ndo_do_ioctl-fallback.patch

```diff
@@ -0,0 +1,27 @@
+From c9701bab45cf3fa244033fcf5e4341a1de627807 Mon Sep 17 00:00:00 2001
+From: Quallenauge <Hamsi2k@freenet.de>
+Date: Mon, 21 May 2018 23:55:01 -0100
+Subject: [PATCH] VGV952CJW33-E-IR: Revert "wireless: wext: remove ndo_do_ioctl
+ fallback" This reverts commit 8bfb3676606454ffec836f56c5dc3e69dfc0956a.
+
+The ralink inic driver currently uses this for controlling the network
+adapters. In future versions the driver has to be modernized to comply
+with future kernel drivers.
```

Comment:
That's already a good example of the patch header I was talking about. Note that the From: should be a real name.